src/components/metadata/styles/metadata-code.scss

```diff
@@ -26,7 +26,7 @@
   padding: 0 variables.$metadata-sidebar-width-open 0 0;
   transform: translateX(100vw);
   transition: transform ease 0.5s 0.1s, left ease 0.5s;
-  z-index: 2;
+  z-index: 4;
```

Comment:
hey I feel like we use z-index everywhere and there should be somewhere to manage them. I've created this ticket here for it https://github.com/kedro-org/kedro-viz/issues/2014.

But for now if you want to create just a variable in the variable file it will be very helpful. Something like `$zindex-metadata-panel`

Reply:
Good idea, I too felt it is hard to know the levels

Reply:
and also i forgot to ask, why "4" but not "3"?

Reply:
When I tested with 3 it had some issue, so thought of keeping it high, but I think I was doing something wrong. Good spot, I changed it to 3 and it works. Also added the variables as suggested. Thank you